code/FulpstationCode/Fulp_changeling_mods/fulp_changeling_procs.dm

```diff
@@ -0,0 +1,15 @@
+/datum/action/changeling/sting/proc/fulp_sting_feedback(mob/user, mob/target)
+	if(!target)
+		return FALSE
+	var/message = "<span class='notice'>We stealthily sting [target.name].</span>"
+
+	if(istype(src, /datum/action/changeling/sting/cryo))
+		to_chat(target, "<span class='warning'>You feel an icy cold prick.</span>")
+		message = "<span class='notice'>We sting [target.name] with our cryostinger.</span>"
+
+	else if(target.mind && target.mind.has_antag_datum(/datum/antagonist/changeling))
+		to_chat(target, "<span class='warning'>You feel a tiny prick.</span>")

```

Comment:
Maybe buff this option to say from whom it came from, since its ling-to-ling

Reply:
Might be a bit much to be honest.